apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch/index.tsx

```diff
@@ -63,11 +65,7 @@ const Launch: React.FunctionComponent< Props > = ( { onSubmit } ) => {
 		}
 	}, [] );
 
-	return (
-		<div className="nux-launch">
-			<CurrentLaunchStep onPrevStep={ handlePrevStep } onNextStep={ handleNextStep } />
-		</div>
-	);
+	return <CurrentLaunchStep onPrevStep={ handlePrevStep } onNextStep={ handleNextStep } />;
```

Comment:
The goal is to have a flattened div to easily have scrollbars with sticky header & footer (and also header & footer that doesn't get overlapped by scrollbars).